File: app/src/main/java/com/apps/widget/LauncherManageScrollView.java
package com.apps.widget;

import android.content.Context;
import android.util.AttributeSet;

/**
 * Backward-compatible name for the management page's clickable-row scroll container.
 */
public class LauncherManageScrollView extends LauncherClickableRowScrollView {

    public LauncherManageScrollView(Context context) {
        super(context);
    }

    public LauncherManageScrollView(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public LauncherManageScrollView(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }
}

File: app/src/main/java/com/apps/widget/LauncherManageScrollView.kt
package com.apps.widget

import android.content.Context
import android.util.AttributeSet

/** Backward-compatible XML name for the management-page scroll container. */
class LauncherManageScrollView @JvmOverloads constructor(
    context: Context,
    attrs: AttributeSet? = null,
    defStyleAttr: Int = 0
) : LauncherClickableRowScrollView(context, attrs, defStyleAttr)
